### PR TITLE
Rename Example to SwifterExample

### PR DESCRIPTION
**Pull request by fortmarek**
Hi 👋 

Resolution of packages fails when _any_ two packages have the same product. That's why having an `Example` without a `Swifter` prefix can quickly lead to conflicts with other packages.

In this PR, I'd like to resolve that by renaming `Example` to `SwifterExample`.

### CODE REVIEW

**Review by elusivestudio — APPROVED** [review]
Looks good!

**Comment by elusivestudio** [comment]
Can we please merge this as it is blocking our project build as well.

**Comment by fortmarek** [comment]
Same story here @elusivestudio. For now we are pointing to the fork I have made for this PR.

File: Xcode/Sources/HttpServerIO.swift
Comment (michaelenger) [inline Xcode/Sources/HttpServerIO.swift]: This is a necessary change, but is unrelated to the renaming of the example directory. Could you please remove this from the PR?

Comment (fortmarek) [reply Xcode/Sources/HttpServerIO.swift]: removed 👍

**Review by michaelenger — APPROVED** [review]
Looks good to me 👍

**Comment by fortmarek** [comment]
> Looks good to me 👍

🎉 I can't merge this, however, so if someone who has enough right could do it, that'd be great 👌

**Pull request merged**
